stylesheets/app.scss

```diff
@@ -201,10 +202,9 @@ button, .button {
     margin-left: 20px;
   }
   &.blue {
-    background:$med-blue;
-    &:hover {
-      background: lighten($med-blue,10%);
-    }
+    background:white;
```

Comment:
Sounds good to me. Might as well leave it in even if it isn't impacting this particular PR.